node/cmd/guardiand/node.go

```diff
@@ -647,6 +651,19 @@ func runNode(cmd *cobra.Command, args []string) {
 			logger.Fatal("Please do not specify --neonContract")
 		}
 	}
+
+	var logPublicRpcDetail common.GrpcLogDetail
```

Comment:
It almost seems like this var should be called "publicRpcLogLevel".

Reply:
Hmm. I think that `publicRpcLogLevel` is not a good name because this variable controls the verbosity of the log entries, not the log level of the log entries.